p2p/discovery/table.go

```diff
@@ -10,14 +10,13 @@ import (
 	"sort"
 
 	"github.com/seeleteam/go-seele/common"
-	"github.com/seeleteam/go-seele/log"
 )
 
 const (
-	alpha      = 3  // Kademlia concurrency factor
+	alpha              = 1 // Kademlia concurrency factor
 	responseNodeNumber = 5 //TODO with this number for test
-	hashBits   = len(common.Hash{}) * 8
-	nBuckets   = hashBits + 1 // Number of buckets
+	hashBits           = len(common.Hash{}) * 8
+	nBuckets           = hashBits + 1 // Number of buckets
```

Comment:
Seems the nBuckets should be hashbits  -1, since we need to exclude current node itself. anything I missed?

Reply:
no, it is the distance value, and distance value is [0,256], which is 257 numbers